src/test/java/org/jitsi/meet/test/web/WebTestBase.java

```diff
@@ -368,6 +368,16 @@ private WebParticipant joinParticipantAndWait(
     {
         WebParticipant participant = joinParticipant(index, meetURL, options);
 
+        // the try catch below is only chrome workaround
+        if (!participant.getType().isChrome())
+        {
```

Comment:
will it be less code if in the try catch below there will be a check at the top: if(!participant.getType().isChrome()) throw ex; ?